src/commands/general/coolpix.ts

```diff
@@ -0,0 +1,35 @@
+import { DiscordApplicationCommandOptionTypes } from "../../../deps.ts";

```

Comment:
This is?

Reply:
My apologies i was trying to use code sandbox and clicked on the wrong thing, please delete.

Reply:
Ahh, no problem.